engines/router/missionctl/config/config.go

```diff
@@ -39,6 +39,15 @@ const (
 	NopLogger ResultLogger = "NOP"
 )
 
+// Protocol is used for router config to indicate spinning up of respective router mission control
+type Protocol string
+
+const (
+	UPI  Protocol = "UPI"
```

Comment:
Do you have objection to make it consistent with Merlin https://github.com/gojek/merlin/blob/969f9d0d7fe7631884498659978e19e75b92bfa1/swagger.yaml#L1669 ?

Reply:
not from me, will do and remove grpc for now (till we have custom grpc)